packages/common/src/constants.ts

```diff
@@ -22,7 +22,7 @@ export enum NETWORK_FAMILY {
   substrate = 'Substrate',
   cosmos = 'Cosmos',
   algorand = 'Algorand',
-  ethereum = 'Ethereum',
+  ethereum = 'evm',
```

Comment:
Are there further implications for this? 

Was it a bug and does it relate to anythign else?